qa/L0_infer/infer_test.py

```diff
@@ -107,25 +107,25 @@ def _infer_exact_helper(tester,
                     input_dtype,
                     output0_dtype,
                     output1_dtype,
-                        output0_raw=output0_raw,
-                        output1_raw=output1_raw,
-                        model_version=model_version,
-                        swap=swap,
-                        outputs=outputs,
-                        use_http=use_http,
-                        use_grpc=use_grpc,
-                        use_http_json_tensors=use_http_json_tensors,
-                        skip_request_id_check=skip_request_id_check,
-                        use_streaming=use_streaming,
-                        correlation_id=correlation_id,
+                    output0_raw=output0_raw,
+                    output1_raw=output1_raw,
+                    model_version=model_version,
+                    swap=swap,
+                    outputs=outputs,
+                    use_http=use_http,
+                    use_grpc=use_grpc,
+                    use_http_json_tensors=use_http_json_tensors,
+                    skip_request_id_check=skip_request_id_check,
+                    use_streaming=use_streaming,
+                    correlation_id=correlation_id,
                     use_system_shared_memory=TEST_SYSTEM_SHARED_MEMORY,
                     use_cuda_shared_memory=TEST_CUDA_SHARED_MEMORY)
 
         input_size = 16
 
         all_ensemble_prefix = ["simple_", "sequence_", "fan_"]
         ensemble_prefix = [""]
-        if ENSEMBLES and ("custom" in BACKENDS):
```

Comment:
Interesting... looks like we were not testing ensembles for non-custom models?

Reply:
For ensemble, you must name the models in the model config, so a lot of ensemble models will need to be generated to cover the backend combinations. And we don't really need to test all combinations as ensemble is backend insensitive.

Reply:
We do have some ensembles that are purely graphdef and savedmodel. We test these under test_ensemble_mix_platform and test_ensemble_mix_type